### PR TITLE
Disable x-scrolling on iOS when iframed

### PR DESCRIPTION
**Pull request by dvoytenko**
b/149328043

TODO:

- [x] Tests

### CODE REVIEW

**Comment by dvoytenko** [comment]
Cleaned up and tests done.

**Comment by kristoferbaxter** [comment]
Looks like two failing tests with `amp-sidebar`. Might want to loop in @wassgha.

**Comment by wassgha** [comment]
Let me manually test otherwise we can skip

**Comment by wassgha** [comment]
Manually works, recommend skipping until we figure it out

**Comment by dvoytenko** [comment]
Thanks @caroqliu for the help with e2e tests.

File: css/ampdoc.css [hunk 1]
```diff
@@ -76,6 +76,23 @@ html.i-amphtml-fie > body {
...
+  overflow-x: hidden !important;
+  overflow-y: visible !important;
+}
+
+/**
+ * iOS jitters when body overflows. See b/149328043. This essentially cancels
+ * `height: auto` from above.
+ */
+html.i-amphtml-singledoc.i-amphtml-iframed:not(.i-amphtml-ios-embed):not([amp4ads]) > body {
+  min-height: calc(100vh - var(--i-amphtml-padding-top, 0px));
```
Comment (jridgewell) [inline css/ampdoc.css]: Why was this added?

Comment (dvoytenko) [reply css/ampdoc.css]: We use to have something like this before for ios-embed mode. Here it's needed again, b/c the `body` loses the `overflowY: visible`. W/o the minimal height, any `position:absolute` element may get cut off.

**Comment by dvoytenko** [comment]
Filed https://bugs.webkit.org/show_bug.cgi?id=209486
